### PR TITLE
Added Search in Sorted rotated array solution(C++)

### PR DESCRIPTION
**Pull request by sukritidawar**
Added the solution in array folder
Issue : #51

### CODE REVIEW

**Comment by sukritidawar** [comment]
@plazzy99 @nakul-19 please review.

File: Array/Search_RotatedSortedArr.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+    int x; //Number we have to search for
+    cin >> x; 
+    cout << search_SortedRotatedArr(arr, x);
+    return 0;
+}
+
+/* Example:
+Input: arr[] = {5, 6, 7, 8, 9, 10, 1, 2, 3}, x = 3
+Output: 8 */
```
Comment (plazzy99) [inline Array/Search_RotatedSortedArr.cpp:64]: 2 test cases required atleast and also mention the space and time complexity here at the bottom

**Review by plazzy99 — COMMENTED** [review]
update the readme for the same

**Comment by sukritidawar** [comment]
Please review @plazzy99 @nakul-19

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
add time and space complexities. make both the testcases similar in terms of format. resolve the conflict.

**Comment by sukritidawar** [comment]
> add time and space complexities. make both the testcases similar in terms of format. resolve the conflict.

I have already added time and space complexities. And I didn't get how to make testcases similar in terms of format. Could you please explain once?

**Comment by yasharth291** [comment]
first resolve the conflicts @sukritidawar

**Pull request closed without merging**